tests/rules/test_hadoop_dfs_missing_dash.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from thefuck.rules.hadoop_dfs_missing_dash import match, get_new_command
+from tests.utils import Command
+
+
+@pytest.mark.parametrize('command', [
+    Command(script='./bin/hdfs dfs ls', stderr='ls: Unknown command\nDid you mean -ls?  This command begins with a dash.'),
```

Comment:
That makes sense. 

Should it then ignore the command and just look for the error message  ": Unknown command" / "Did you mean ?"?